apis/config/v1beta1/configuration_types.go

```diff
@@ -146,6 +146,9 @@ type ControllerMetrics struct {
 	// metrics will be reported.
 	// +optional
 	EnableClusterQueueResources bool `json:"enableClusterQueueResources,omitempty"`
+
+	// +optional
+	EnableLocalQueueMetrics bool `json:"enableLocalQueueMetrics,omitempty"`
```

Comment:
I very much agree, especially when it comes to passing parameters

Reply:
There was a comment about increasing [cardinality](https://github.com/kubernetes-sigs/kueue/issues/1833#issuecomment-1994473006) and wanting to leave this behind a long term config field

Reply:
I see, but in that case I would like to go via the KEP process. Pity the comment does not mention why cardinality is a problem - is it for usability (this could be solved by aggregation), or performance. Do you have some other references why cardinality might be a problem in k8s. 

I assume we don't have many more LQs than namepaces, which also let me check what we do in the core k8s. I see that we have metrics depending on Namespace, [example](https://github.com/kubernetes/kubernetes/blob/810e9e212ec5372d16b655f57b9231d8654a2179/pkg/registry/core/pod/rest/metrics.go#L38C4-L38C13). However, in this case we use explicitly CounterOpts.Namespace. Maybe we could also do it this way? PTAL. 

If you want this feature in 0.10 I think the only chance is a short KEP, don't change API, and guard it by Alpha feature gate (disabled by default). Then for second iteration of alpha investigate if we need the API switch.

Reply:
`https://kubernetes.io/docs/reference/instrumentation/metrics/`

There's a few metrics here with a namespace label.

There's an existing [KEP](https://github.com/kubernetes-sigs/kueue/pull/2516) that I was having a bit of trouble implementing since it included the ability to use namespace/`local_queue` selectors for metric collection.

Reply:
I would suggest to update the KEP and hide the mretics behind alpha feature gate. This will not impact users /customers and we don't commit to maintain the API. Then as graduation point for beta we re-evaluate both approaches

Reply:
Sure, but we will not be able to do this for 0.10. Let's update the KEP and revisit the knobs for 0.11 before Beta.

Reply:
I'm ok with v0.11 due to the tighter schedule for v0.10, and this feature is still alpha.